src/pybind/mgr/volumes/fs/stats_util.py

```diff
@@ -235,18 +244,24 @@ def _update_progress_bars(self):
             self.finish()
             return
 
+        # there has to be 1 ongoing clone for this method to run, perhaps it
+        # wasn't found by it because the index entry for it hasn't been created
+        # yet.
+        if self.ongoing_clones_count == 0:
+            self.ongoing_clones_count = 1
+
         # onpen bar (that is progress bar for clone jobs in ongoing and pending
         # state) is printed when clones are in pending state. it is kept in
         # printing until all clone jobs finish.
-        show_onpen_bar = True if len(clones) > self.max_concurrent_clones \
+        show_onpen_bar = True if len(clones) > self.ongoing_clones_count \
```

Comment:
is this a typo ?
what is **onpen bar** ?

Reply:
sorry, noticed the preceding comment about "ongoing and pending" clones which is what **onpen** means